commitlint.config.js

```diff
@@ -1,3 +1,45 @@
+const scopes = [
+  'a11y',
+  'accordion',
+  'alert',
+  'badge',
+  'build',
+  'button',
+  'card',
+  'checkbox',
+  'datagrid',
+  'date-picker',
+  'dropdown',
+  'form',
+  'grid',
+  'header',
+  'i18n',
+  'input',
```

Comment:
We could use a scope for general stuff. Looking at this list, I don't know what I would put the CSS work under. It doesn't really apply to build and touches about 18 other scopes. Also anything specific to base or any refactor that spans across more than one component. It would be nice to have a scope for those...

Reply:
If something bridges over many aspects, I don't think you'd put in a scope. The CSS work is a good example. If it isn't isolated to one scope then no scope. Thoughts?

Reply:
Right, I think scopes are meant to be optional and just help identify for specific issues. If not specific then no scope is needed. example `refactor: update scss vars to use css custom props`

Reply:
But the rule on line 43. Does it always expect a scope? Or is that just saying that if there is a scope it always has to be in this enum?

Reply:
It will only validate a scope against this list when it is provided, commits with no scopes are allowed.